typings/midi-defs-a8a84257afa70721ae462448048a87301fc1554a.d.ts

```diff
@@ -133,7 +133,7 @@ interface MIDI {
   setController: (
     channelId: number, type: Object, value: Object, delay: number) => void;
   setDefaultPlugin: (midi: Obejct) => void;
```

Comment:
Object* (Not related to this PR changes!)

Reply:
Can we do something to avoid such typo in these files?

Reply:
done! There is an issue filed for this. (#10219)